samples/js-shopping-cart/cassandra-store.yaml

```diff
@@ -6,9 +6,9 @@ kind: StatefulStore
 metadata:
   name: cassandra
 spec:
-
   # This is a Cassandra deployment, so supply configuration for Cassandra.
-  cassandra:
-
+  type: Cassandra
+  deployment: Unmanaged
```

Comment:
This was just changed recently for the new operator:

https://github.com/cloudstateio/cloudstate/pull/441/files#diff-bd08628341d8901dc8500e3f54739f4dL13-L20